airbyte-webapp/src/pages/OnboardingPage/components/FirstSuccessfulSync/FirstSuccessfulSync.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { FormattedMessage } from "react-intl";
+
+import { Text } from "components/base/Text";
+
+import styles from "./FirstSuccessfulSync.module.scss";
+
+export const FirstSuccessfulSync: React.FC = () => {
+  return (
+    <div className={styles.container}>
```

Comment:
Can we give this directly a `data-testid="firstSuccessfulSync"` just in case we later need to filter on it in tests or one of our growth tools to trigger other stuff?

Reply:
Sure 😄